ambry-router/src/main/java/com/github/ambry/router/CompressionService.java

```diff
@@ -306,6 +296,17 @@ public ByteBuf compressChunk(ByteBuf chunkByteBuf, boolean isFullChunk, boolean
       return null;
     }
 
+    // Compress accepted, emit metrics. MB/sec = (#bytes/#microseconds) x BytesPerMicroSecondsToMBToSec
```

Comment:
LGTM, just wanna know what will be the value of these metrics if we never hit this point and return from the catch-block ?

Reply:
If not hitting this point, we don't have emit metrics.  So the metrics in the compressed blob requests.

Reply:
So the value is 0 or just a gap in the graph ?